src/Uno.UI/FeatureConfiguration.cs

```diff
@@ -135,12 +135,7 @@ public static class Control
 			/// Will never be deferred when .ApplyTemplate() is called explicitly.
 			/// More information there: https://github.com/unoplatform/uno/issues/3519
 			/// </remarks>
-			public static bool UseDeferredOnApplyTemplate { get; set; }
-#if __ANDROID__ || __IOS__ || __MACOS__
-			// opt-in for iOS/Android/macOS
-#else
-				= true;
-#endif
+			public static bool UseDeferredOnApplyTemplate { get; set; } = true;
```

Comment:
As this is fixing an issue, can we add a test that replicates the twopaneview issue?

Reply:
Seems a bit fails did happen 😅

Reply:
Yes. It's quite expected it won't go very smooth 😄 

Trying to figure it out 😕